### PR TITLE
Drop Ubuntu 16.04 and 18.04 support

### PR DESCRIPTION
**Pull request by zilchms**
includes: #150

### CODE REVIEW

File: manifests/repository.pp [hunk 1]
```diff
@@ -122,7 +122,7 @@
         $url = "${scheme}${address}"
       }
       if $os  == 'Ubuntu' {
-        unless $osrelease in ['12.04', '14.04', '16.04', '18.04', '20.04'] {
+        unless $osrelease in ['20.04'] {
```
Comment (bastelfreak) [inline manifests/repository.pp:125]: I think this is bad practice. IMO we should remove the whole check and people should rely on the metadata.json to identify if their OS is supported or not.

Comment (zilchms) [reply manifests/repository.pp:125]: that is a fair point, thought about it too. I dont know anything about bareos specifically, but is the location that gets built even really that specific and only works on LTS? But in the end it doesnt matter, I agree with you we should remove that whole check-fail construct

Comment (zilchms) [reply manifests/repository.pp:125]: there are a few more lets say odd choices in this module. I think I will clean those up in another PR

**Comment by zilchms** [comment]
hmm, I am not really a fan of deleting the check. On the other hand: it only ever ran on ubuntu18.04 anyways. At the same time: I think the reload command for non systemd OSes is broken since forever (or relaod is somehow not a typo)

**Pull request merged**